### PR TITLE
feat: new command mass upload (DEV-1626)

### PR DESCRIPTION
**Pull request by jnussbaum**
resolves DEV-1626, DEV-1784

### CODE REVIEW

File: src/dsp_tools/utils/xml_upload_enhanced.py [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    Args:
+        batch: list of paths to image files
+        sipi_port: port number of SIPI
+
+    Returns:
+        None
+    """
+    print(f"Process batch with ID {id}...")
+    for imgpath in batch:
+        response = requests.post(url=f'http://localhost:{sipi_port}/upload', files={'file': open(imgpath, 'rb')})
```
Comment (jnussbaum) [inline src/dsp_tools/utils/xml_upload_enhanced.py:83]: @irinaschubert Perhaps you can help here. I always get a 400 error "Token missing". Vij says that after commenting out the passage in `sipi/scripts/upload.lua`, this error shouldn't happen any more, but it persists. To see what we comment out in `sipi/scripts/upload.lua`, see in this PR the file `xmlupload-enhanced.md`, line 51

Comment (jnussbaum) [reply src/dsp_tools/utils/xml_upload_enhanced.py:83]: I found it out. I had a "dsp-api" repo on my desktop that I left untouched, and a "dsp-api-modified" where I modified the contents. But when starting the Docker container, it took some data and configuration from the "dsp-api" folder. Now that I renamed these two folders, it works.

Comment (irinaschubert) [reply src/dsp_tools/utils/xml_upload_enhanced.py]: cool :)

File: src/dsp_tools/utils/xml_upload_enhanced.py [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+    """
+    # retrieve list of all derivatives 
+    upload_candidates: list[str] = []
+    upload_candidates.extend(glob.glob(f"{Path(sipi_processed_path)}/{Path(internal_filename).stem}/**/*.*"))
+    upload_candidates.extend(glob.glob(f"{Path(sipi_processed_path)}/{Path(internal_filename).stem}*.*"))
+
+    # make a plausibility check
+    min_num_of_candidates = 5 if orig_file_path.suffix == ".mp4" else 3
+    if len(upload_candidates) < min_num_of_candidates:
+        raise BaseError(f"Local SIPI created only the following files for {orig_file_path}, which is too less: {upload_candidates}")
```
Comment (irinaschubert) [inline src/dsp_tools/utils/xml_upload_enhanced.py:191]:
```suggestion
        raise BaseError(f"Local SIPI created only the following files for {orig_file_path}, which is too little: {upload_candidates}")
```

**Review by irinaschubert — CHANGES_REQUESTED** [review]
I'm not done yet with the review but I already wanted to let you know that we wanted to test something today with this version of the branch and it failed. Apparently, Sipi looks for a wrong path (the local one instead of the remote one). If you do an enhanced XML upload you can see it.

**Review by irinaschubert — COMMENTED** [review]
Another bug we found: when trying to create an ontology locally, dsp-tools can't connect to dsp-api.

**Pull request closed without merging**